shell-model/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: ATOMIC STRUCTURE 
+uuid: shell-model
+prerequisites:
+  - none 
```

Comment:
Maybe `periodic-table` is a good prerequisite?

Reply:
I'll add.